tests/test_models.py

```diff
@@ -185,6 +185,25 @@ def test_custom_tms_bounds_user_crs():
     assert custom_tms.bounds(0, 0, 0) == (-120, 30, -110, 40)
 
 
+def test_custom_tms_decimation():
+    """Check bounds with epsg6342 and custom decimation base."""
+    extent = (238170, 4334121, 377264, 4473215)
+    left, bottom, right, top = extent
+    for decimation_base in [2, 3, 4, 5]:
+        custom_tms = TileMatrixSet.custom(
+            extent,
+            pyproj.CRS.from_epsg(6342),
+            decimation_base=decimation_base,
+        )
```

Comment:
can we add `assert not custom_tms.is_quadtree` 🙏

Reply:
done!